src/loader/loader.c

```diff
@@ -220,6 +220,9 @@ extension_mark_loader_present()
 	void	  **presentptr = find_rendezvous_variable(RENDEZVOUS_LOADER_PRESENT_NAME);
 
 	*presentptr = &loader_present;
+
+	/* This is purely for backwards compatibility for versions 0.9.1 and before. */
+	SetConfigOption("timescaledb.loader_present", "on", PGC_USERSET, PGC_S_SESSION);
```

Comment:
This is unfortunate. We got rid of this GUC and now need it again for backwards compatibility. Is there no other way?

Reply:
Can we set it only when loading these versions?